tapgarden/planter.go

```diff
@@ -1978,12 +1985,26 @@ func (c *ChainPlanter) prepAssetSeedling(ctx context.Context,
 	case c.pendingBatch != nil:
 		log.Infof("Adding %v to existing MintingBatch", req)
 
+		// The batch already exist and may contain seedlings. Before
+		// adding the seedling to the batch, we'll ensure that the
+		// batch will still be valid after adding the seedling.
+		err := c.pendingBatch.ValidateSeedling(*req)
+		if err != nil {
+			return fmt.Errorf("seedling can not be added to "+
+				"pending batch: %w", err)
+		}
+
 		c.pendingBatch.Seedlings[req.AssetName] = req
 
+		// The batch may already exist, but it may not have any
```

Comment:
Could mention that this would happen if a batch got funded before having seedlings added.